Kernel_23/doc/Kernel_23/Kernel_23.txt

```diff
@@ -498,8 +498,8 @@ in a type-safe way \cgal uses return values of type
 `boost::optional< boost::variant< T...  > >` were `T...` is a
 list of all possible resulting geometric objects.  The exact result
 type of an intersection can be determined through the metafunction
-`cpp11::result_of<Kernel::Intersect_2(Type1, Type2)>` or
-`cpp11::result_of<Kernel::Intersect_3(Type1, Type2)>`, where
+`CGAL::cpp11::result_of<Kernel::Intersect_2(Type1, Type2)>` or
```

Comment:
Actually, the reason behind that is to add `CGAL` where the <name> is part of an example or it is a part that is not included in the CGAL namespace and to remove `CGAL` where the <name> is already in the CGAL namespace. In the end, we will probably change the docs anyway afterwards so it will all be cleaned up one more time.